src/Compilers/VisualBasic/Portable/Emit/EditAndContinue/EmitHelpers.vb

```diff
@@ -34,6 +34,9 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Emit
             Dim serializationProperties = compilation.ConstructModuleSerializationProperties(emitOpts, runtimeMDVersion, baseline.ModuleVersionId)
             Dim manifestResources = SpecializedCollections.EmptyEnumerable(Of ResourceDescription)()
 
+            Dim updatedMethods = ArrayBuilder(Of MethodDefinitionHandle).GetInstance()
```

Comment:
I was confused by this as these don't look used (aside from ToImmutableAndFree()). Is there more work coming later?

Reply:
They're passed to `SerializeToDeltaStreams` on line 86. On the off chance this comment was meant for another location, there is another spot that probably has these arrays and there are changes coming in future, but the API is only in -vs-deps.

Reply:
I think I failed to use one of the following correctly:

1. My eyes
2. My code review tool

😄